test/cql-pytest/README.md

```diff
@@ -1,4 +1,4 @@
-Single-node functional tests for Scylla's CQL features.
+# Single-node functional tests for Scylla's CQL features.
```

Comment:
The commit message title says one thing, the rest of the commit message says another thing.
Please change the title, the commit message, and/or split the patch.

Reply:
I split it into two commits